tools/bq-notifier/main_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This code is a prototype and not engineered for production use.
+// Error handling is incomplete or inappropriate for usage beyond
+// a development sample.
+
+package bqnotifier
+
+import (
+	"context"
+	"io/ioutil"
+	"log"
+	"testing"
+)
+
+func TestJson(*testing.T) {
```

Comment:
We want to return test errors using the testing.T reference that is available. So we would do something like this..

func TestJson(t *testing.T) {
   ... setup and run the test ...
  if err != nil {
    t.Errorf("'%s' failed: HandleJobComplete(%v, %v) = (%v) want (nil)", name, input1, input2, err)
  }
}